proto/api_test.go

```diff
@@ -150,10 +150,10 @@ func TestCombinable(t *testing.T) {
 
 func TestSetGoErrorCopy(t *testing.T) {
 	rh := ResponseHeader{}
-	err := &Error{Message: "test123"}
-	rh.SetGoError(err)
-	err.Message = "321tset"
-	if rh.Error.Message != "test123" {
+	oErr := &Error{Message: "test123"}
+	rh.Error = oErr
+	rh.SetGoError(&testError{})
```

Comment:
I think the old test is incorrect, actually. Take a look at https://github.com/cockroachdb/cockroach/pull/533: It's really supposed to exercise the line that I mentioned above. The new test does that, the old doesn't.